src/actions/PrescriptionActions.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-unused-expressions */
+
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+import { batch } from 'react-redux';
+
+import { UIDatabase, createRecord } from '../database';
+
+import { PrescriberActions } from './PrescriberActions';
+import { WizardActions } from './WizardActions';
+
+export const PRESCRIPTION_ACTIONS = {
+  REFRESH: 'Prescription/refresh',
+};
+
+export const updateDirection = (id, newValue) => (_, getState) => {
```

Comment:
This goes for all of these functions - can't see why then need to be exported

Reply:
thanks! 405f7fc